bool use_guided
bool initialize
bool no_push_calc
---
# Replace with PushVector
Header header
geometry_msgs/Point start_point
float64 push_angle
float64 push_dist
uint32 num_objects
uint32 object_idx
uint32 object_id
bool no_push
uint32[] singulated
uint32 push_bin
